easyfix-messages/src/fields/basic_types.rs

```diff
@@ -922,7 +932,7 @@ impl UtcTimeOnly {
     /// input's precision is adjusted to requested one
     pub fn with_millis(time: NaiveTime) -> UtcTimeOnly {
         UtcTimeOnly {
-            timestamp: time.with_nanosecond(time.nanosecond() / 1_000_000).unwrap(),
```

Comment:
Why did you removed input adjustment here and in `with_micros`?

It may look not necessary from serialization perspective, but this value is also available via `pub fn timestamp()`.